terratest/utils/test_settings.go

```diff
@@ -25,7 +25,10 @@ type TestSettings struct {
 	Port1           int
 	DNSServer2      string
 	Port2           int
+	DNSServer3      string
```

Comment:
So basically this just fixes number of clusters to 3? You can't run more without modyfing terratest code?

Reply:
no, you can deploy as many as you like w/ `make deploy-full-local-setup CLUSTERS_NUMBER=7` but in terratests for round robin 3 clusters are created and some testing is done (start them all, then scale podinfo to 0 on one and check if the IPs of the other two clusters are correctly propagated to gslb's cr's status, etc.)